docs/Using-Fleet/REST-API.md

```diff
@@ -3523,6 +3523,10 @@ These API endpoints are used to automate MDM features in Fleet. Read more about
 - [Get Apple Push Notification service (APNs)](#get-apple-push-notification-service-apns)
 - [Get Apple Business Manager (ABM)](#get-apple-business-manager-abm)
 - [Turn off MDM for a host](#turn-off-mdm-for-a-host)
+- [Upload a bootstrap package](#upload-a-bootstrap-package)
+- [Get metadata about a bootstrap package](#get-metadata-about-a-bootstrap-package)
+- [Delete a bootstrap package](#delete-a-bootstrap-package)
+- [Download a bootstrap package](#download-a-bootstrap-package)
```

Comment:
I think the new filter is missing from the docs, in List Hosts but probably also Count Hosts, Lists Hosts in Label, Count hosts in label? (assuming it applies in all those places, I believe most of the times this is the case)

Reply:
And also the new `macos_setup` object added to the response of Get Host.